### PR TITLE
skip tests reaching outside network for disconnected cluster

### LINKED ISSUE #9535: skip noobaa related tests for disconnected mode cluster
skip noobaa related tests for disconnected mode cluster. This cluster type is not able to create resources that trying to reach outside network.
- test_namespace_store_creation_rules
~- test_blocking_pool_creation_rules~

### PR DESCRIPTION
**Pull request by DanielOsypenko**
Fixes #9535

### CODE REVIEW

File: tests/functional/ui/test_error_improvements.py [hunk 1]
```diff
@@ -112,6 +114,7 @@ def test_namespace_store_creation_rules(
     @bugzilla("2193109")
     @polarion_id("OCS-4873")
     @skipif_hci_provider_or_client
+    @skipif_disconnected_cluster
```
Comment (sagihirshfeld) [inline tests/functional/ui/test_error_improvements.py:117]: It doesn't look like `test_blocking_pool_creation_rules` is reaching the outside network - please remove this skip decorator.

Comment (DanielOsypenko) [reply tests/functional/ui/test_error_improvements.py]: you're absolutely right 💯 
removed

**Review by sagihirshfeld — APPROVED** [review]

**Review by am-agrawa — COMMENTED** [review]
Aren't we skipping `test_blocking_pool_creation_rules` as part of this PR?

**Comment by DanielOsypenko** [comment]
@am-agrawa updated #https://github.com/red-hat-storage/ocs-ci/pull/9536 `test_blocking_pool_creation_rules` is not reaching network, so dropped decorator for this test

**Comment by am-agrawa** [comment]
> @am-agrawa updated ##9536 `test_blocking_pool_creation_rules` is not reaching network, so dropped decorator for this test

You meant #9535 . Looks good, thanks!

**Review by am-agrawa — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**